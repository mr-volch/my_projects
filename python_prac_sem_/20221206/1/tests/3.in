7,11,2,12
