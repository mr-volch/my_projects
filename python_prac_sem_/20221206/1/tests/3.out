0_7
0_11
1_7
2_7
1_11
3_7
2_11
4_7
5_7
3_11
6_7
4_11
